libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/Plugins/SFSmartStore/SFSmartStorePlugin.m

```diff
@@ -343,8 +343,11 @@ - (void)pgReIndexSoup:(CDVInvokedUrlCommand *)command
 - (void)pgShowInspector:(CDVInvokedUrlCommand *)command
 {
     [self runCommand:^(NSDictionary* argsDict) {
-        SFSmartStoreInspectorViewController* inspector = [self inspector:command];
-        [inspector present:self.viewController];
+        __weak typeof(self) weakSelf = self;
+        __block SFSmartStoreInspectorViewController *inspector = [self inspector:command];
```

Comment:
This doesn't need to be `__block`.  Its value doesn't change inside the block where it's used.